langchain/llms/base.py

```diff
@@ -34,6 +55,38 @@ def _get_verbosity() -> bool:
     return langchain.verbose
 
 
+def create_base_retry_decorator(
+    error_types: List[Type[BaseException]], max_retries: int = 1
+) -> Callable[[Any], Any]:
+    """Create a retry decorator for a given LLM and provided list of error types."""
+    import logging
```

Comment:
this and the logger initialization can go at top of file i think

Reply:
Done